llvm_ir_dataset_utils/analysis_tools/MAKEFILE

```diff
@@ -0,0 +1,26 @@
+
+CC := clang
```

Comment:
Can you only set `CC` if it is not already set to `clang`, maybe with a warning? Otherwise it might be difficult to point the script at a specific compiler without having to install it as default.